contact.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <link rel="stylesheet" href="/src/output.css" />
+      <title>Contact</title>
+   </head>
+   <body class="font-outfit text-webGreen">
+      <!-- Navbar -->
+      <header>
+         <nav
+            class="top-0 z-10 sticky flex flex-row justify-between items-center lg:mb-12 md:px-8 p-4 lg:p-6 xl:p-6 font-outfit"
+         >
+            <div class="flex flex-col md:text-2xl lg:text-2xl">
```

Comment:
I think, this md:text-2xl will also work on the larger screen, you don't need to add another one